manim/cli/new/group.py

```diff
@@ -0,0 +1,189 @@
+    """Updates the manim.cfg file after reading it from the project_cfg_path.
+
+    Parameters
+    ----------
+    cfg : :class:`dict`
+        values used to update manim.cfg found project_cfg_path.
+    project_cfg_path : :class:`Path`
+        Path of manim.cfg file.
+    """
+    config = configparser.ConfigParser()
+    config.read(project_cfg_path)
+    cli_config = config["CLI"]
+    for key, value in cfg_dict.items():
+        if key == "resolution":
+            cli_config["pixel_height"] = str(value[0])
+            cli_config["pixel_width"] = str(value[1])
+        else:
+            cli_config[key] = str(value)
+
+    with open(project_cfg_path, "w") as conf:
+        config.write(conf)
+
+
+@click.command(
+    context_settings=CONTEXT_SETTINGS,
+    epilog=EPILOG,
+)
+@click.argument("project_name", type=Path, required=False)
+@click.option(
+    "-d",
+    "--default",
+    "default_settings",
+    is_flag=True,
+    help="Default settings for project creation.",
+    nargs=1,
+)
+def project(default_settings, **args):
+    """Creates a new project.
+
+    PROJECT_NAME is the name of the folder in which the new project will be initialized.
+    """
+    if args["project_name"]:
+        project_name = args["project_name"]
+    else:
+        project_name = click.prompt("Project Name", type=Path)
+
+    # in the future when implementing a full template system. Choices are going to be saved in some sort of config file for templates
+    template_name = click.prompt(
+        "Template",
+        type=click.Choice(get_template_names(), False),
+        default="Default",
+    )
+
+    if project_name.is_dir():
+        console.print(
+            f"\nFolder [red]{project_name}[/red] exists. Please type another name\n"
+        )
+    else:
+        project_name.mkdir()
+        new_cfg = dict()
+        new_cfg_path = Path.resolve(project_name / "manim.cfg")
+
+        if not default_settings:
+            for key, value in CFG_DEFAULTS.items():
+                if key == "scene_names":
+                    new_cfg[key] = template_name + "Template"
+                elif key == "resolution":
+                    new_cfg[key] = select_resolution()
+                else:
+                    new_cfg[key] = click.prompt(f"\n{key}", default=value)
+
+            console.print("\n", new_cfg)
+            if click.confirm("Do you want to continue?", default=True, abort=True):
+                copy_template_files(project_name, template_name)
+                update_cfg(new_cfg, new_cfg_path)
+        else:
+            copy_template_files(project_name, template_name)
+            update_cfg(CFG_DEFAULTS, new_cfg_path)
+
+
+@click.command(
+    context_settings=CONTEXT_SETTINGS,
+    no_args_is_help=True,
+    epilog=EPILOG,
+)
+@click.argument("scene_name", type=str, required=True)
+@click.argument("file_name", type=str, required=False)
+def scene(**args):
+    """Inserts a SCENE to an existing FILE or a new FILE
```

Comment:
```suggestion
    """Inserts a SCENE to an existing FILE or creates a new FILE.
```

Reply:
I was planning on adding documentation for these in the future.

thanks a bunch for the review